src/patternfly/components/Table/examples/Table.md

```diff
@@ -218,6 +218,51 @@ cssPrefix: pf-c-table
   {{/table-tbody}}
 {{/table}}
 ```
+
+```hbs title=Sortable-with-headers-that-wrap
+{{#> table table--id="table-sortable-wrap" table--grid="true" table--modifier="pf-m-grid-lg" table--attribute='aria-label="This is a sortable table example with headers that wrap"'}}
+  {{#> table-thead}}
+    {{#> table-tr}}
+      {{#> table-th table-th--attribute='scope="col"' table-th--sortable="true" table-th--sortable-wrap="true" table-th--selected="true" table-th--asc="true"}}
```

Comment:
Then you could just add `table-th--modifier="pf-m-wrap"` here instead of needing updated structure.

Reply:
+1, we use `table-th--modifier="pf-m-wrap"` on the other includes, probably good to be consistent instead of using `table-th--sortable`